test/regress/CMakeLists.txt

```diff
@@ -2513,10 +2514,19 @@ set(regression_disabled_tests
   regress1/sygus/interpol_from_pono_3.smt2
   regress1/sygus/interpol_dt.smt2
   regress1/sygus/inv_gen_fig8.sy
-  # rely on heuristic solution reconstruction TODO #3146 revisit
+  # new algorithm is slow at reconstructing random constants
+  regress0/sygus/c100.sy
+  # terms require rewrites not in database and are too big to enumerate
+  regress0/sygus/aig-si.sy
+  regress0/sygus/parity-AIG-d0.sy
   regress1/sygus/array_search_2.sy
   regress1/sygus/array_sum_2_5.sy
   regress1/sygus/crcy-si-rcons.sy
+  regress1/sygus/parity-si-rcons.sy
+  # returns wrong answer
```

Comment:
Can you clarify this a bit more?  What options return a wrong answer?

Reply:
Fixed. Regression test passes now.